src/Sil/Bundle/ContactBundle/Resources/views/form/address-form.html.twig

```diff
@@ -0,0 +1,84 @@
+                <h3>New Address</h3>
+            </div>
+        </div>
+    </h1>
+    {% if form.vars.value and form.vars.value.id is defined %}
+        {% set updateAction =  path('sil_contact_address_edit', {'id': form.vars.value.id, 'list': true}) %}
+    {% else %}
+        {% set updateAction = '' %}
+    {% endif %}
+    {{ form_start(form, {
+        'attr': {
+            'id': 'address-form',
+            'class': 'ui form',
+            'data-action': path('sil_contact_address_create', {'list': true}),
+            'data-edit-action': updateAction
+        }
+    }) }}
+    <div class="ui stackable center aligned grid content">
+        <div class="row">
+            <div class="eight wide column">
+                <div class="field">
+                    {{ form_label(form.street) }}
+                    {{ form_errors(form.street) }}
+                    {{ form_widget(form.street) }}
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="four wide column">
+                <div class="field">
+                    {{ form_label(form.city.name, 'City') }}
+                    {{ form_errors(form.city.name) }}
+                    {{ form_widget(form.city.name) }}
+                </div>
+            </div>
+            <div class="four wide column">
+                <div class="field">
+                    {{ form_label(form.city.postCode) }}
+                    {{ form_errors(form.city.postCode) }}
+                    {{ form_widget(form.city.postCode) }}
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="four wide column">
+                <div class="field">
+                    {{ form_label(form.province) }}
+                    {{ form_errors(form.province) }}
+                    {{ form_widget(form.province) }}
+                </div>
+            </div>
+            <div class="four wide column">
+                <div class="field">
+                    {{ form_label(form.country) }}
+                    {{ form_errors(form.country) }}
+                    {{ form_widget(form.country, { 'attr': {'class': 'ui dropdown'} }) }}
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="four wide column">
+                <div class="field">
+                    {{ form_label(form.type) }}
+                    {{ form_errors(form.type) }}
+                    {{ form_widget(form.type, { 'attr': {'class': 'ui dropdown'} }) }}
+                </div>
+            </div>
+        </div>
+
+        {{ form_widget(form.contact, {'value': ''}) }}
+    </div>
+    <div class="ui divider"></div>
+    <div class="content center aligned">
+        {{ form_widget(form.save, { 'attr': {'class': 'ui big teal button'} }) }}
+        <div id="cancel-address" class="ui big orange button">Cancel</div>
+    </div>
+    {{ form_end(form) }}
```

Comment:
Missing call to `form_rest` in order to display missing fields (e.g. field `_token`)

Reply:
form_end calls form_rest already